webui/react/src/components/kit/Button.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button as AntdButton } from 'antd';
+import React, { CSSProperties, MouseEvent, ReactNode } from 'react';
+
+interface ButtonProps {
+  children?: ReactNode;
+  className?: string;
+  danger?: boolean;
+  disabled?: boolean;
```

Comment:
i think its more maintainable
```suggestion
  import type { ButtonProps as AntdButtonProps } from 'antd';
  
  disabled?: AntdButtonProps['disabled'];
```

or if we know props we wanna hide, we can do `Omit<AntdButtonProps, 'something`>

Reply:
For now we want to explicitly specify which props we're using. I think this will make it easier for people to use this component in the future since it's clear which props are available.

Reply:
It also increases maintainability by creating an interface that doesn't change when we update antd.